zone/spawn2.cpp

```diff
@@ -485,7 +486,8 @@ bool ZoneDatabase::PopulateZoneSpawnList(uint32 zoneid, LinkedList<Spawn2*> &spa
 		"animation "
 		"FROM "
 		"spawn2 "
-		"WHERE zone = '%s' AND  (version = %u OR version = -1)",
+		"WHERE TRUE {} AND zone = '{}' AND (version = {} OR version = -1) ",
```

Comment:
What does `TRUE` do here?

Reply:
It’s for ANDing, you don’t know if content filter criteria will return anything so if it doesn’t the query will fail with a syntax error before AND. True is like a no-op placeholder so we can always AND regardless

Reply:
👍